### DISCUSSION TITLE
Welcome to vscode-extension-for-zowe Discussions!

### DISCUSSION THREAD

**Discussion by zdmullen** [zowe/zowe-explorer-vscode#1104] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zdmullen** [comment]
This should be something that we can utilize when our discussions in particular Issues or Pull Requests become particularly detailed...

**Comment by MikeBauerCA** [comment]
Hello, within Zowe, I have experience with Zowe CLI, Zowe Explorer, and the emerging suite of Zowe SDKs. I am also involved in the Open Mainframe Project's [COBOL Programming Course]( https://github.com/openmainframeproject/cobol-programming-course). If you are also interested in discussing the Zowe CLI, feel free to add to [our discussions](https://github.com/zowe/zowe-cli/discussions) as well!
